ui/src/views/tools/ManageInstances.vue

```diff
@@ -322,8 +323,8 @@
             <a-col v-if="!isDiskImport" :md="24" :lg="(!isMigrateFromVmware && showManagedInstances) ? 12 : 24">
               <a-card class="instances-card">
                 <template #title>
-                  {{ $t('label.unmanaged.instances') }}
-                  <a-tooltip :title="$t('message.instances.unmanaged')">
+                  {{ (isMigrateFromVmware && vmwareVcenterType === 'existing') ? $t('label.instances') : $t('label.unmanaged.instances') }}
+                  <a-tooltip :title="(isMigrateFromVmware && vmwareVcenterType === 'existing') ? $t('message.instances.migrate.vmware') : $t('message.instances.unmanaged')">
```

Comment:
shouldn't isMigrateFromVmware be enough condition? i.e.
```suggestion
                  {{ isMigrateFromVmware ? $t('label.instances') : $t('label.unmanaged.instances') }}
                  <a-tooltip :title="isMigrateFromVmware ? $t('message.instances.migrate.vmware') : $t('message.instances.unmanaged')">
```

Reply:
@DaanHoogland current change allows showing `Instances` label only when the vCenter is an existing one. External vCenter will always considered to have unmanaged instances

Reply:
yes but the label doesn't have to change for that, that's my point.